src/redux/appointments/appointmentsSlice.js

```diff
@@ -0,0 +1,76 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { cancelAppointment, getAllAppointments, scheduleAppointment } from './appointmentsApi';
+import storage from '../../app/localStorage';
+
+const baseURL = process.env.REACT_APP_BASE_URL;
```

Comment:
- Here, it looks like I need to add a `.env` file to continue running the application, I kindly invite you to add all specific instructions on the readme file, because this is a piece of important information, and is a good practice to add it there.